jung-api/src/test/java/edu/uci/ics/jung/graph/PackageSanityTests.java

```diff
@@ -31,16 +31,20 @@ public PackageSanityTests() {
     setDistinctValues(TreeBuilder.class, TREE_BUILDER_A, TREE_BUILDER_B);
     setDistinctValues(CTree.class, TREE_A, TREE_B);
     setDefault(Optional.class, Optional.empty());
+    ignoreClasses(clazz -> Objects.equals(clazz, TestUtil.class));
   }
 
-  @Override
```

Comment:
I'm just eyeballing this, but since the `testNulls` below still has an `@Override`, shouldn't this one retain it, too?

Reply:
Oops, yep you're right! I'll submit a quick PR to use `@Override` consistently.

Reply:
Done. :)